packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -42,18 +46,46 @@ export default (props) => {
     ),
     customProps.className,
   );
+  
+  if (customProps.className) {
+    mainClasses = `${mainClasses} ${customProps.className}`;
+  }
 
   delete customProps.className;
 
+  let profilePhoto;
+
+  if (avatar && avatar.alt && avatar.initials) {
+    profilePhoto = (
+      <Avatar
+        className={cx('profile-photo')}
+        alt={avatar.alt}
+        color={avatar.color}
+        hashValue={avatar.hashValue}
+        image={avatar.image}
+        initials={avatar.initials}
+        isAriaHidden={avatar.isAriaHidden}
+        isDeceased={!!deceasedDate}
+      />
+    );
+  } else if (photo) {
+    /*
+     * This condition is added to keep the profile photo changes passive to Demographics banner.
```

Comment:
Could you log a github issue to make this change?

Reply:
Issue Logged. I'll add the issue in the comment.